Update Azure DevOps Oauth configuration instructions (#3036)
codenvy/che-docs: modules/administration-guide/partials/proc_applying-the-microsoft-azure-devops-services-oauth-app-secret.adoc

:_content-type: PROCEDURE
:description: Applying the Microsoft Entra ID OAuth App Secret
:keywords: microsoft-azure-devops-entra-id-oauth-app, apply, secret
:navtitle: Applying the Microsoft Entra ID OAuth App Secret
// :page-aliases:

[id="applying-the-microsoft-entra-id-oauth-app-secret"]
= Applying the Microsoft Entra ID OAuth App Secret

> Prepare and apply the Microsoft Entra ID Secret.

Reviewer: Abstract restates title. Add WHY.

```suggestion
Prepare and apply the Secret that enables {prod-short} to authenticate with Microsoft Entra ID for Azure DevOps repository access.
```

.Prerequisites

> * Setting up the Microsoft Entra ID OAuth App is completed.

Reviewer: Prerequisites state conditions that already exist. Use declarative "You have..." form.

```suggestion
* You have set up the Microsoft Entra ID OAuth App.
```
